CONTRIBUTING.md

```diff
@@ -823,20 +823,17 @@ rejecting any PR. Although [anyone can review PRs](#reviews), only the
 acknowledgement (or "ack") from an Approver counts towards the approval of a PR.
 
 Approvers are listed in GitHub teams, one for each repository. The project
-uses the [Pull Approve](https://PullApprove.com) service meaning each
```

Comment:
Ah, is this the drop of pullapprove that I mentioned in a previous review commit? In which case, although you could merge to avoid the spell-correct-text-I-then-delete, it's not a biggie.

Reply:
It's difficult to get the balance. If I'd munged it all into a single commit someone would complain ;) I separated the pullapprove changes as they probably deserve to stand out from the rest: we should have cleaned up the docs immediately after switching off pullapprove of course.